OpenRA.Mods.Common/Effects/RallyPointIndicator.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace OpenRA.Mods.Common.Effects
 {
-	class RallyPointIndicator : IEffect, IEffectAboveShroud, IEffectAnnotation
+	public class RallyPointIndicator : IEffect, IEffectAboveShroud, IEffectAnnotation
```

Comment:
How is this change related to disabling upnp on local games?

Reply:
Not at all, but the commit message explains: `Allow mod code to set rally point indicators.` I figured @Mailaender just wanted to get this easy fix in without needing to open another PR.

Reply:
See https://github.com/OpenRA/OpenRA/pull/18673 for reference.